python/private/common/py_runtime_rule.bzl

```diff
@@ -186,10 +207,11 @@ runtime. For a platform runtime this attribute must not be set.
 """,
         ),
         "interpreter": attr.label(
-            allow_single_file = True,
+            allow_files = True,
```

Comment:
Please put a comment about why this is allow_files instead of allow_single_file, as normally we'd expect an attribute that is supposed to be a single file to be marked as such.